packages/edit-widgets/src/store/selectors.js

```diff
@@ -238,5 +238,5 @@ export const canInsertBlockInWidgetArea = createRegistrySelector(
  * @return {boolean} Is active.
  */
 export function __unstableIsFeatureActive( state, feature ) {
-	return get( state.preferences.features, [ feature ], false );
+	return get( state.preferences, [ 'features', feature ], false );
```

Comment:
I tightened this up a bit as the tests threw and error when there was no `features` property.

Probably an edge case as the reducer sets one up.

Reply:
What kind of edge case it is? I would like to know more :)

Reply:
`__.get()` will get the value at path (the second arg) of object. 

So if `state.preferences` doesn't exist the console was throwing an error when we tried to access `state.preferences.features`.

I've never seen this happen of course, but just in case the state is, for some reason empty, I guess it doesn't hurt to be more defensive.

That's only my opinion! Totally happy to revert if folks think I'm being paranoid. :)

Reply:
> but just in case the state is, for some reason empty, I guess it doesn't hurt to be more defensive.

I disagree. Most of the times when we're building a user-facing software, we would want the bug to appear as early as possible, so that we can fix it before it's too late or hidden behind dozens of levels. If, for some reasons the state were empty, we would want it the throw an error, preferably during development. So that we can spot the error earlier and debug it easier too. If we hide it or swallow the error, then we'll never know when something went wrong, possibly in a more horrible way 😅.

Hence, I would suggest to not prematurely guard it unless we really have a reason to :). Does it make sense to you? This is also just my opinion though!

Reply:
> If, for some reasons the state were empty, we would want it the throw an error, preferably during development. So that we can spot the error earlier and debug it easier too. If we hide it or swallow the error, then we'll never know when something went wrong, possibly in a more horrible way

Thanks for explaining your reasoning, and helping my thinking!

The error would throw if the reducer wasn't properly initialized, so you're right now that I've had time to think about it: we'd want to be able trace the cause back to the reducer rather than have no feedback at all. 

I'll tidy this up.